src/mumble/UserInformation.cpp

```diff
@@ -129,7 +129,14 @@ void UserInformation::update(const MumbleProto::UserStats &msg) {
 
 		const MumbleProto::Version &mpv = msg.version();
 
-		qlVersion->setText(tr("%1 (%2)").arg(Version::toString(mpv.version())).arg(u8(mpv.release())));
+		Version::mumble_raw_version_t version = Version::UNKNOWN;
+		if (mpv.has_version_2()) {
+			version = mpv.version_2();
+		} else if (mpv.has_version()) {
+			version = Version::fromLegacyVersion(mpv.version());
```

Comment:
In this case, we could even show a warning if the converted version appears as if it was suffering from the >255 issue

Reply:
In the UI - after all `UserInformation` is a UI anyway, so I would simply integrate a note somewhere in there